src/core/solutions/bench/brown_function.h

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright (c) 2018, Jairo Rojas-Delgado <jrdelgado@uci.cu>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+* Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+* Neither the name of the <organization> nor the
+names of its contributors may be used to endorse or promote products
+derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+#ifndef DNN_OPT_CORE_SOLUTIONS_BENCH_BROWN_FUNCTION
+#define DNN_OPT_CORE_SOLUTIONS_BENCH_BROWN_FUNCTION
+
+#include <core/base/generator.h>
+#include <core/base/solution.h>
+
+namespace dnn_opt
+{
+namespace core
+{
+namespace solutions
+{
+namespace bench
+{
+
+/**
+ * @brief The brown class represents an optimization solutions which
+ * fitness cost is calculated via Brown function.
+ *
+ * The equation for this function is given by:
+ *
+ * f(x) = \sum_{i=0}^{n-1}{x_i^{2*{x_i+1}+2}}+{x_i+1}^{2*{{x_i}^2}+2}
+ *
+ * Brown function have a global minima in {0,..., 0} with a value of 0.
+ * A commonly used search domain for testing is [-1, 4]. Brown is continuous, 
+ * differentiable, non-separable, scalable and unimodal. See the following 
+ * reference [f_25] in:
+ * 
+ * MOMIN, JAMIL; YANG, Xin-She. A literature survey of benchmark functions for 
+ * global optimization problems. Journal of Mathematical Modelling and Numerical 
+ * Optimisation, 2013, vol. 4, no 2, p. 150-194.
+ *
+ *
+ * @author Jairo Rojas-Delgado <jrdelgado@uci.cu>
+ * @version 1.0
+ * @date November, 2016
```

Comment:
@alejandrom247 here you need to update your name and email. Also, remember to update the corresponding date (October, 2019). Yo will need yo go through each new class you have included.